static/app/utils/useOwners.tsx

```diff
@@ -0,0 +1,72 @@
+import {useEffect, useMemo} from 'react';
+
+import {useMembers} from 'sentry/utils/useMembers';
+import {useTeams} from 'sentry/utils/useTeams';
+import {useTeamsById} from 'sentry/utils/useTeamsById';
+
+interface Options {
+  /**
+   * The current selected values that will be ensured loaded. These should be
+   * in the actor identifier format
+   */
+  currentValue?: string[];
+}
+
+/**
+ * Hook to fetch owner options
+ */
+export function useOwners({currentValue}: Options) {
+  // Ensure the current value of the fields members is loaded
+  const ensureUserIds = useMemo(
+    () =>
+      currentValue?.filter(item => item.startsWith('user:')).map(user => user.slice(7)),
+    [currentValue]
+  );
+  useMembers({ids: ensureUserIds});
+
+  const {
+    members,
+    fetching: fetchingMembers,
+    onSearch: onMemberSearch,
+    loadMore: loadMoreMembers,
+  } = useMembers();
```

Comment:
why does the hook need to be listed twice?

Reply:
lol good question. This is how it is in the other component. Let me see if this is not required